pipeline/trusted-resources/pkg/config/config.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	cm "knative.dev/pkg/configmap"
+)
+
+// Config holds the collection of configurations that we attach to contexts.
+// Configmap named with "config-trusted-resources" should include "signing-secret-path" as key in the data
+type Config struct {
+	// CosignKey defines the value of the key in configmap data
+	CosignKey string
+}
+
+const (
+	// CosignPubKey is the value of the key in configmap data
+	CosignPubKey = "signing-secret-path"
```

Comment:
Want to make sure I understand correctly: do you mean CosignPubKey is the name of the key in configmap data here? other than the value, i.e. data[CosignPubKey]?

If so, the comment is kind of misleading :), you may want to update it.

Reply:
yes, data[CosignPubKey] is correct.Maybe "name of the key"